x-pack/solutions/security/plugins/security_solution/public/flyout/document_details/left/components/attach_to_active_timeline.test.tsx

```diff
@@ -49,7 +49,7 @@ describe('AttachToActiveTimeline', () => {
     );
 
     expect(getByTestId(SAVE_TIMELINE_BUTTON_TEST_ID)).toBeInTheDocument();
-    expect(getByTestId(SAVE_TIMELINE_BUTTON_TEST_ID)).toHaveStyle('background-color: #FEC514');
+    expect(getByTestId(SAVE_TIMELINE_BUTTON_TEST_ID).className).toContain('warning');
```

Comment:
Just curious about these changes, I'm guessing these colours were cleared out when we wrapped the test provider with `EuiProvider`?

Reply:
Exactly! The component had no color value